downstream/modules/navigator/proc-review-config-tui.adoc

```diff
@@ -0,0 +1,99 @@
+
+[id="proc-review-config-tui_{context}"]
+
+
+
+= Reviewing your Ansible configuration from the {Navigator} textual user interface
+
+[role="_abstract"]
+
+You can review your Ansible configuration with the {Navigator} textual user interface in interactive mode and delve into the settings.
+
+.Prerequisites
+
+* A valid Ansible configuration file.
+
+.Procedure
+
+
+. Start {Navigator}
++
+```
+$ ansible-navigator
+```
+
+. Review the Ansible configuration.
++
+```
+ :config
+
+ OPTION                                  DEFAULT SOURCE  VIA                             CURRENT VALUE
+  0│ACTION_WARNINGS                      True default default                         True
+  1│AGNOSTIC_BECOME_PROMPT               True default default                         True
+  2│ALLOW_WORLD_READABLE_TMPFILES        True default default                         False
+  3│ANSIBLE_CONNECTION_PATH              True default default                         None
+  4│ANSIBLE_COW_ACCEPTLIST               False env    ANSIBLE_COW_ACCEPTLIST          ['bud-frogs', 'dragon']
+  5│ANSIBLE_COW_PATH                     True default default                         None
+  6│ANSIBLE_COW_SELECTION                True default default                         default
+  7│ANSIBLE_FORCE_COLOR                  True default default                         False
+  8│ANSIBLE_NOCOLOR                      True default default                         False
+  9│ANSIBLE_NOCOWS                       True default default                         False
+ 10│ANSIBLE_PIPELINING                   True default default                         False
+ 11│ANY_ERRORS_FATAL                     True default default                         False
+ 12│BECOME_ALLOW_SAME_USER               True default default                         False
+ 13│BECOME_PLUGIN_PATH                   True default default                         ['/home/runner/.ansible/plugins/become', '/usr/share/ansible/plugins/become']
+ 14│CACHE_PLUGIN                         False env    ANSIBLE_CACHE_PLUGIN            jsonfile
+ 15│CACHE_PLUGIN_CONNECTION              False env   ANSIBLE_CACHE_PLUGIN_CONNECTION /runner/artifacts/d2e5510d-b518-4625-8067-5a4da2a1d32c/fact_cac
+ 16│CACHE_PLUGIN_PREFIX                  True default default                         ansible_facts
+
+```
+
```

Comment:
maybe a note that some values reflect settings inside the execution environment necessary for the execution environment to function.  These will show as non-default, but may not be set in the user's ansible config file